pulp_ansible/app/tasks/collections.py

```diff
@@ -493,7 +493,6 @@ async def _add_collection_version(self, api_version, collection_version_url, met
             d_artifacts=[d_artifact],
             extra_data=extra_data,
         )
-        self.parsing_metadata_progress_bar.increment()
         await self.put(d_content)
```

Comment:
Here I think we need to return 1 + number of dependencies synced (from line 471). Basically we need to sum up the return values from all gathers() to accurately update the amount of collection versions synced. 
```
num_synced = sum(await asyncio.gather(*tasks))
self.parsing_metadata_progress_bar.increase_by(num_synced)
```